LibSource/Resource.h

```diff
@@ -0,0 +1,211 @@
+#ifndef __RESOURCE_STORAGE_H__
+#define __RESOURCE_STORAGE_H__
+
+#include <stdint.h>
+#include <cstring>
+#include "OpenWareMidiControl.h"
+#include "ServiceCall.h"
+#include "ProgramVector.h"
+#include "FloatArray.h"
+
+
+/*
+ * This class should not be instantiated directly, but it can be returned by untyped 
+ * getResource() calls to storage.
+ */
+
```

Comment:
comment out of date?

Reply:
Yes! I will re-read them just in case.

Also, some of the methods may end up unnecessary (i.e. setData). It was inherited from time when saving resource was available. It may still be useful for saving presets from MIDI callback, but for now we may remove it.